_posts/2017-10-18-understanding-colourblindness.md

```diff
@@ -1,43 +1,37 @@
 ---
 layout: post
-title: "Understanding colourblindness"
-description: "What is colour blindness, and how can we design with it in mind?"
+title: "Understanding colorblindness"
+description: "What is color blindness, and how can we design with it in mind?"
 author: matthew_higgins
```

Comment:
one last thing, Emily.

You've done some good work here and you should get credit for it.  Please add:
`updated_by: emily_lane` on a new line after "author"

and also update the [authors yml file](https://github.com/a11yproject/a11yproject.com/blob/gh-pages/_data/authors.yml) with your information (using "emily_lane" to match the line I mentioned above") and commit to this PR.

Sorry, I didn't notice this in my previous reviews.